pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -330,6 +335,10 @@ func TestToKubeContainerStatusWithResources(t *testing.T) {
 		},
 	} {
 		t.Run(desc, func(t *testing.T) {
+			if test.os != runtime.GOOS {
```

Comment:
same here, could we change to something like:
```go 
if runtime.GOOS == "windows" {
    t.Skipf("skipping test %q because it is not intended for windows", desc)
}
```
This way we skip the test only for windows